freebridge-ai/main.py

```
@@ -1,79 +1,73 @@
+            | prompt
+            | llm
+            | StrOutputParser()
+        )
         
-    except (ConnectionError, ValueError) as ce: 
-        logger.error(f"Connection/Config Error during init: {ce}")
-    except Exception as e: 
-        logger.exception(f"Unexpected error during initialization: {e}")
-    
+        logger.info("RAG System Initialized.")
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)